Firestore/core/src/firebase/firestore/immutable/sorted_map_iterator.h

```diff
@@ -189,4 +189,5 @@ class SortedMapIterator {
 }  // namespace firestore
 }  // namespace firebase
 
+// NOLINTNEXTLINE
```

Comment:
What's the lint error with the next line?

Reply:
I think the issue is likely that the internal cpplint doesn't see this as a header guard because it does not match the path.

If you added a copybara rule that prepended FIRESTORE_CORE_SRC_ with the equivalent symbol translated for the actual path this would likely just work. You can try this with manual edits to see if it satisfies the linter.